person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
```

Comment:
- [ ] `READEME.md` file: To make your documentation looks more professional and descriptive, kindly ensure that you update some of your sections by adding correct and appropriate information for each one, such as:
  - Add your project title with a good and meaningful description
![Screenshot from 2023-07-26 03-14-07](https://github.com/enmanuelbayen/OOP-school-library/assets/83042742/a220e75c-583e-46d0-8153-1244563051a9)

  - Update some of the sub-sections of the `Getting Started` main section with correct information and For this I suggest you refer **[this gist_file](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)** to help you hoe to proceed.